src/script/interpreter.cpp

```diff
@@ -61,6 +61,75 @@ static inline void popstack(std::vector<valtype>& stack)
     stack.pop_back();
 }
 
+static inline void push4_le(std::vector<valtype>& stack, uint32_t v)
+{
+    auto v_le = htole32(v);
+    valtype vch(reinterpret_cast<unsigned char*>(&v_le), reinterpret_cast<unsigned char*>(&v_le) + 4);
```

Comment:
emplace_back.

Reply:
For my understanding, 

The difference between
```
valtype vch(args);
stack.push_back(std::move(vch));
```
and
```
stack.emplace_back(args);
```
is just a cheap move operation.

Reply:
The `emplace_back` seems unreadable to my eyes, so I tend to prefer the second one.